src/intl/locale.ts

```diff
@@ -8,11 +8,15 @@ export const DEFAULT_LOCALE = 'en';
 // We use pseudo locale used for localization testing, to reliably determine if we need to fallback to the default newsroom language
 export const DUMMY_DEFAULT_LOCALE = 'qps-ploc';
```

Comment:
What are the `__BRAND__` properties in this file for?

_(can't place a comment on the actual line for some reason)_

Reply:
It is a workaround to create a nominal type https://github.com/Microsoft/Typescript/issues/202
These two string are not compatible, so i can't assign one to another

Reply:
Is that extra type safety really needed, considering you added the `isGlobal` property on the `LocaleObject` class?

Reply:
I think yes, because they are similar and I think it is good to check them on type level

Reply:
I'm still not sure this is needed, they can be distinguished as well with variable naming or JSDoc comments.

Reply:
Changed https://github.com/prezly/theme-kit-nextjs/pull/215/commits/40ef543d010db69a2cad48d1b77481c111e005b7